percona-xtradb-cluster-5.7-backup/backup.sh

```diff
@@ -149,8 +149,14 @@ backup_s3() {
 	{ set +x; } 2>/dev/null
 	s3_add_bucket_dest
 	set -x
-	is_object_exist "$S3_BUCKET" "$S3_BUCKET_PATH.$SST_INFO_NAME" || xbcloud delete ${INSECURE_ARG} $XBCLOUD_EXTRA_ARGS --storage=s3 --s3-bucket="$S3_BUCKET" "$S3_BUCKET_PATH.$SST_INFO_NAME"
-	is_object_exist "$S3_BUCKET" "$S3_BUCKET_PATH" || xbcloud delete ${INSECURE_ARG} $XBCLOUD_EXTRA_ARGS --storage=s3 --s3-bucket="$S3_BUCKET" "$S3_BUCKET_PATH"
+
+	if [[ -z $SKIP_FAILED_BACKUP_CLEANUP ]]; then
```

Comment:
@pooknull please add this var as an example into CR https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L667